components/aws/sagemaker/tests/unit_tests/tests/test_train.py

```diff
@@ -86,6 +88,16 @@ def test_create_training_job(self):
     )
     self.assertEqual(response, 'test-job')
 
+  def test_stop_training_job(self):
```

Comment:
you can rename this to "test_main_stop_training_job()" as this is testing the main() and not the "_utils.stop_training_job()"